source/fluid/image_view.d

```diff
@@ -120,6 +132,26 @@ class ImageView : Node {
 
             io.assertTexture(root.texture, Vector2(0, 0), color!"fff");
 
+            version (Have_raylib_d) {
```

Comment:
Unit tests run without Raylib. This code won't compile.

Reply:
That sounds like a real problem. You should try to get them working with and without so that the Raylib backend is tested.

Reply:
Actually, the unittests are _definately_ running with Raylib. I didn't even do anything to the `dub.json`. Just running `dub test` gives me some Raylib outputs in the terminal.